page-factory-api/src/main/java/ru/sbtqa/tag/pagefactory/environment/Environment.java

```diff
@@ -10,46 +10,46 @@
  */
 public class Environment {
 
-    private static DriverService driverService;
-    private static PageActions pageActions;
-    private static PageChecks pageChecks;
-    private static Reflection reflection;
+    static InheritableThreadLocal<DriverService> driverService = new InheritableThreadLocal<DriverService>();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace the type specification in this constructor call with the diamond operator ("<>"). ([squid:S2293](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS2293))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=126&issues=AWeIHOFkcrQ7oSxgxngD&open=AWeIHOFkcrQ7oSxgxngD)

[ISSUE_KEY]: AWeIHOFkcrQ7oSxgxngD